src/downshift.js

```diff
@@ -983,6 +994,10 @@ class Downshift extends Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
+    if (this.getMenuProps.called && !this.getMenuProps.suppressRefError) {
```

Comment:
This should probably be:

```javascript
if (
  this.getMenuProps.called &&
  !this.getMenuProps.suppressRefError &&
  preval`module.exports = process.env.BUILD_REACT_NATIVE !== 'true'`
) {
```

so this code can be eliminated for native. Same with the above.

Reply:
I’ll fix that tomorrow! Hmm any reason why we dont do that for ’getRootProps’ as well? Im not so familiar with RN but why wouldnt we want to validate the ref?

Reply:
Yeah, I just checked and the only place we're using `_rootNode` is for codepaths that react native doesn't need. So we could probably do the same thing for those.

I'm thinking about making [a custom macro](https://github.com/kentcdodds/babel-plugin-macros) that will shorten things for us/make these kinds of branches easier to maintain. But that'll be for later :)

Reply:
Oh okay! I've pushed the latest changes now, and I think you should be able to use `--autosquash` to squash them together to one commit

I like the sound of that idea!